jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/coder/OdeMapDataCreatorHelperTest.java

```diff
@@ -1,38 +1,36 @@
 package us.dot.its.jpo.ode.coder;
 
 import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.fail;
-
-import org.junit.jupiter.api.Test;
-
-import com.fasterxml.jackson.databind.JsonNode;
 
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.Test;
 import us.dot.its.jpo.ode.model.OdeMapData;
-import us.dot.its.jpo.ode.util.XmlUtils;
-import us.dot.its.jpo.ode.util.XmlUtils.XmlUtilsException;
 
+/**
```

Comment:
Mainly checkstyle updates to this unit test. Moving sample data to dedicated files and javadocs.